metals/src/main/scala/scala/meta/internal/metals/debug/BuildTargetClasses.scala

```diff
@@ -46,6 +47,13 @@ final class BuildTargetClasses(
         .map(_.fullyQualifiedName)
     )
 
+  def jvmRunEnvironment(
```

Comment:
Is the separate method needed?

Reply:
I wanted to make this variable private and add a method since `TrieMap` is mutable, but maybe you're right and we can remove it